lib/pos_blockchain.go

```diff
@@ -291,13 +301,51 @@ func (bc *Blockchain) updateCurrentView(desoBlock *MsgDeSoBlock) {
 
 func (bc *Blockchain) GetUncommittedTipView() (*UtxoView, error) {
 	// Connect the uncommitted blocks to the tip so that we can validate subsequent blocks
-	panic("GetUncommittedTipView: Not implemented yet")
+	highestCommittedBlock, committedBlockIndex := bc.getHighestCommittedBlock()
+	if highestCommittedBlock == nil {
+		// TODO: do we just want to connect all the blocks in best chain then?
```

Comment:
This should never happen. IMO, you don't have to worry about this edge case. Returning an error is sufficient